### ISSUE TITLE
vmhgfs compile error on 4.1.0

### ISSUE THREAD

**Issue by rrva** [rasa/vmware-tools-patches#61]
Using kernel build system.
make -C /lib/modules/4.1.0-2-generic/build/include/.. SUBDIRS=$PWD SRCROOT=$PWD/. \
  MODULEBUILDDIR= modules
make[1]: Entering directory '/usr/src/linux-headers-4.1.0-2-generic'
  CC [M]  /tmp/vmhgfs/vmhgfs-only/backdoorGcc64.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/message.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/dir.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/rpcout.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/hgfsUtil.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/cpName.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/link.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/request.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/stubs.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/page.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/hgfsEscape.o
  CC [M]  /tmp/vmhgfs/vmhgfs-only/file.o
/tmp/vmhgfs/vmhgfs-only/file.c:172:18: error: 'new_sync_read' undeclared here (not in a function)
    .read       = new_sync_read,
                  ^
/tmp/vmhgfs/vmhgfs-only/file.c:173:18: error: 'new_sync_write' undeclared here (not in a function)
    .write      = new_sync_write,
                  ^
scripts/Makefile.build:258: recipe for target '/tmp/vmhgfs/vmhgfs-only/file.o' failed
make[2]: *** [/tmp/vmhgfs/vmhgfs-only/file.o] Error 1
Makefile:1395: recipe for target '_module_/tmp/vmhgfs/vmhgfs-only' failed
make[1]: *** [_module_/tmp/vmhgfs/vmhgfs-only] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-4.1.0-2-generic'
Makefile:120: recipe for target 'vmhgfs.ko' failed
make: *** [vmhgfs.ko] Error 2

**Comment by rrva** [comment]
This seems relevant: https://github.com/torvalds/linux/commit/5d5d568975307877e9195f5305f4240e506a2807

**Comment by patmans** [comment]
taviso's patch worked fine for me.

**Comment by Passw** [comment]
taviso's patch worked for me.

```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu Wily Werewolf (development branch)
Release:        15.10
Codename:       wily
$ uname -a
Linux sun 4.1.0-3-generic #3-Ubuntu SMP Tue Jul 28 12:22:42 UTC 2015 i686 i686 i686 GNU/Linux
```

**Comment by rasa** [comment]
Fixed by #64, presumably.
